src/machines/java/com/enderio/machines/common/blockentity/base/PoweredMachineEntity.java

```diff
@@ -268,21 +244,7 @@ public boolean canAct() {
 
     // endregion
 
-    // region Capabilities and Serialization
-
-    @Override
-    public <T> LazyOptional<T> getCapability(Capability<T> cap, @Nullable Direction side) {
```

Comment:
This was removed because it was simply not required turns out; idk who added this or why (was probably me during one of the million rewrites)

Reply:
Is this not the way other mods interact with our caps? This is how we expose caps no?

Reply:
Capability providers are doing this for us; https://github.com/SleepyTrousers/EnderIO-Rewrite/blob/9e37ca9b2abb1728990fc890dd71fbb94ad60820/src/core/java/com/enderio/core/common/blockentity/EnderBlockEntity.java#L237